pkg/models/plan_discussion.go

```diff
@@ -8,37 +8,60 @@ import (
 type PlanDiscussion struct {
 	baseStruct
 	modelPlanRelation
-	Content      string           `json:"content" db:"content"`
-	Status       DiscussionStatus `json:"status" db:"status"`
-	IsAssessment bool             `json:"isAssessment" db:"is_assessment"`
+	Content            string           `json:"content" db:"content"`
+	Status             DiscussionStatus `json:"status" db:"status"`
+	IsAssessment       bool             `json:"isAssessment" db:"is_assessment"`
+	CreatedByUserInfo  UserInfo         `json:"createdByUserInfo" db:"created_by_user_info"`
+	ModifiedByUserInfo *UserInfo        `json:"modifiedByUserInfo" db:"modified_by_user_info"`
```

Comment:
I think it might end up being less overall code to not have these 2 fields added to the model, but just let GQL resolvers fill them out